frontend/packages/knative-plugin/src/topology/components/side-panel/knative-eventsource-tab-sections.tsx

```diff
@@ -0,0 +1,19 @@
+import * as React from 'react';
+import { GraphElement } from '@patternfly/react-topology';
+import { referenceFor } from '@console/internal/module/k8s';
+import TopologySideBarTabSection from '@console/topology/src/components/side-bar/TopologySideBarTabSection';
+import EventSourceResources from '../../../components/overview/EventSourceResources';
+import { CamelKameletBindingModel } from '../../../models';
+import { isDynamicEventResourceKind } from '../../../utils/fetch-dynamic-eventsources-utils';
+
+export const getKnativeSidepanelSinkSection = (element: GraphElement) => {
+  const { obj } = element.getData().resources;
+  if (isDynamicEventResourceKind(referenceFor(obj)) || obj.kind === CamelKameletBindingModel.kind) {
+    return (
+      <TopologySideBarTabSection>
+        <EventSourceResources obj={obj} ownedSources={element.getData().resources.eventSources} />;
```

Comment:
Nit: obj can be reused here?
```suggestion
        <EventSourceResources obj={obj} ownedSources={obj.eventSources} />;
```

Reply:
@debsmita1 wait sorry this might not be correct, I didn't realize `obs` was destructured.